Pipeline/pipeline.py

```diff
@@ -475,85 +495,123 @@ def validation(self, x, y):
         """
         self.test = y
         if self.scaler1 is not None:
-            X_scaled = self.scaler1.transform(x.values.reshape(-1, len(x.columns)))
+            x_scaled = self.scaler1.transform(x.values.reshape(-1, len(x.columns)))
         if self.scaler2 is not None:
             self.train = pd.DataFrame(
-                self.scaler2.inverse_transform(self.model.predict(X_scaled)),
+                self.scaler2.inverse_transform(self.model.predict(x_scaled)),
                 columns=y.columns,
             )
         else:
-            self.train = pd.DataFrame(self.model.predict(X_scaled), columns=y.columns)
+            self.train = pd.DataFrame(self.model.predict(x_scaled), columns=y.columns)
+
         return self.train, self.test
 
     def save_model(self, name):
-        """
+        """Save a neural netowork model.
 
         Parameters
         ----------
         name : str
-            DESCRIPTION.
+            The folder's name where the model will be saved
 
+        Examples
+        --------
         """
-        if not os.path.isdir(name):
-            os.makedirs(name)
-        self.model.save(r"{}/{}.h5".format(name, name))
-        dump(self.y.columns, open(r"{}/{}_columns.pkl".format(name, name), "wb"))
-        dump(self.best, open(r"{}/{}_best_features.pkl".format(name, name), "wb"))
-        if self.scaler1 != None:
-            dump(self.scaler1, open(r"{}/{}_scaler1.pkl".format(name, name), "wb"))
-        if self.scaler2 != None:
-            dump(self.scaler2, open(r"{}/{}_scaler2.pkl".format(name, name), "wb"))
-
-
-class Model:
+        path = Path(__file__).parent / name
+        if not path.exists():
+            path.mkdir()
+
+        self.model.save(path / r"{}.h5".format(name, name))
+        dump(self.y.columns, open(path / r"{}_columns.pkl".format(name), "wb"))
+        dump(self.best, open(path / r"{}_best_features.pkl".format(name), "wb"))
+        dump(self.x.columns, open(path / r"{}_features.pkl".format(name), "wb"))
+        if self.scaler1 is not None:
+            dump(self.scaler1, open(path / r"{}_scaler1.pkl".format(name), "wb"))
+        if self.scaler2 is not None:
+            dump(self.scaler2, open(path / r"{}_scaler2.pkl".format(name), "wb"))
+
+
+class Model(object):
     def __init__(self, name):
         self.name = name
 
     def load_model(self):
-        self.model = load_model(r"{}/{}.h5".format(self.name, self.name))
-        self.columns = load(
-            open(r"{}/{}_columns.pkl".format(self.name, self.name), "rb")
-        )
+        """Load a neural netowork model from rossml folder.

```

Comment:
Typo netowork.

Reply:
I also suggest using only `load` instead of `load_model`.

Reply:
Done!